.sonarcloud.properties

```diff
@@ -0,0 +1 @@
+
```

Comment:
Sonar files should not be added to the repository.

Reply:
Removed the unnecessary file.